native/src/main/java/org/ballerinalang/redis/connection/RedisConnectionCommandExecutor.java

```diff
@@ -85,4 +98,22 @@ public void close() throws RedisConnectorException {
             connManager.getRedisCommands().quit();
         }
     }
+
+    public BArray clusterInfo() throws RedisConnectorException {
+        if (!connManager.isClusterConnection()) {
+            throw new RedisConnectorException("Cannot execute cluster info command on a non-cluster connection");
+        }
+
+        RedisClusterCommands<?, String> clusterCommands = null;
+        try {
+            clusterCommands = (RedisAdvancedClusterCommands<?, String>) connManager.getRedisClusterCommands();
```

Comment:
```suggestion
        try {
            RedisClusterCommands<?, String> clusterCommands = (RedisAdvancedClusterCommands<?, String>) connManager.getRedisClusterCommands();
```

Reply:
Can't move into the `try` block as this is used inside the `final` clause